setup.py

```diff
@@ -24,7 +24,7 @@ def find_version(*file_paths):
 
 
 install_requires = [
-    'botocore==1.35.40',
```

Comment:
This file should not be touched. You'll need to remove this from your commit. You can do this with some variation of:

```
git reset HEAD^ -- path/to/file
git commit --amend
```